alacritty/src/renderer/mod.rs

```diff
@@ -302,16 +299,22 @@ impl GlyphCache {
         })
     }
 
+    /// Clear currently cached data in both GL and the registry.
+    pub fn clear_glyph_cache<L: LoadGlyph>(&mut self, loader: &mut L) {
+        loader.clear();
+        self.cache = HashMap::default();
+        self.cursor_cache = HashMap::default();
+
+        self.load_common_glyphs(loader);
+    }
+
     pub fn update_font_size<L: LoadGlyph>(
         &mut self,
         font: config::Font,
         dpr: f64,
         loader: &mut L,
     ) -> Result<(), font::Error> {
-        // Clear currently cached data in both GL and the registry
-        loader.clear();
-        self.cache = HashMap::default();
-        self.cursor_cache = HashMap::default();
+        self.clear_glyph_cache(loader);
```

Comment:
There's no reason why this needs to be at the top, right?

That's why my previous review just suggested to call `clear_glyph_cache` directly. No point to do the work twice.